e2etest/rewarding_test.go

```diff
@@ -471,6 +472,78 @@ func TestBlockEpochReward(t *testing.T) {
 	}
 }
 
+func TestClaimReward(t *testing.T) {
+	t.Skip("")
```

Comment:
why skip the test?
You can refer to changes in `integrity_test.go`, need to register a poll protocol (so that it will return a valid candidate, whose rewardAddr you can claim reward from it)